tests/test_ticket11419.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of afw.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+
+import lsst.afw.table as afwTable
+import lsst.utils.tests
+
+
+class SchemaOffsetTest(unittest.TestCase):
+
+    def setUp(self):
+        self.schema = afwTable.Schema()
+        self.schema.addField('test1', type='ArrayF', size=370010025)
+        self.schema.addField('test2', type='ArrayF', size=370010025)
```

Comment:
I guess this covers case when size of two fields is larger than 2GB. Maybe add a case when single field exceeds 2GB too?